src/components/dialogs/generator-creation-dialog.js

```diff
@@ -436,6 +477,28 @@ const GeneratorCreationDialog = ({
         handleClose();
     };
 
+    const withVoltageRegulationInputs = () => {
+        return (
+            <>
+                {gridItem(voltageRegulationTypeField, 4)}
+                <Box sx={{ width: '100%' }} />
+                <Grid item xs={4} justifySelf={'end'} />
+                {gridItem(voltageSetpointField, 4)}
+                <Box sx={{ width: '100%' }} />
+                {voltageRegulationType === 'DISTANT' && (
```

Comment:
Maybe compare to enum value here (see previous comment)

Reply:
Done